### PR TITLE
Show members of relative costume in Create_Member

### PR DESCRIPTION
**Pull request by AsukaCHikaru**
TODO: 
memebr_gen sort
click action

### CODE REVIEW

File: src/components/Create_Group.js [hunk 1]
```diff
@@ -1,102 +1,132 @@
...
-  handleSelectCos = (event) => {
-    this.setState({cosSelected: event.target.value})
-  }
-  
+const mapDispatchToProps = {
+  createPhotosGroup,
+  createPhotosCostume
+};
+
+class Create_Group extends Component {
```
Comment (dwi2) [inline src/components/Create_Group.js]: I have a suggestion. We are going to have a lot of `CreateXXX` components. How about lets put them all under `components/create/` folder? 
And also the naming is not consistent with other components like `PhotoList`, `PhotoItem`.
Can we spare the `_`? Just `CreateGroup` should be fine.

File: src/constants/cos_mem_map.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export const cos_mem_map = {
+  imaizumi_yui: ['2019_july_2_yukata', '2019_july_3_safari', ],
+  shida_manaka: ['2019_july_2_yukata', ],
+  sugai_yuuka: ['2019_july_2_yukata', '2019_july_3_safari', ],
+  hirate_yurina: ['2019_july_2_yukata', '2019_domesticTour', '2019_domesticTour', ],
+  moriya_akane: ['2019_july_2_yukata', '2019_july_3_safari', ],
+  inoue_rina: ['2019_july_2_yukata', ],
+  seki_yumiko: ['2019_july_2_yukata', '2019_july_3_safari', '2019_july_5_check', ],
+  yamasaki_ten: ['2019_july_2_yukata', '2019_july_5_check', ],
+};
```
Comment (dwi2) [inline src/constants/cos_mem_map.js]: need a new line here.

File: src/constants/members.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    },
+    {
+      member_name_en: 'yamasaki_ten',
+      member_name: '山﨑天',
+      member_name_gana: 'やまさき_てん',
+      member_gen: 2,
+      member_graduated: false
+    },
+  ],
+};
```
Comment (dwi2) [inline src/constants/members.js]: need a new line here.

File: src/style/Create_Member.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+          }
+        }
+
+        & h6{
+          font-weight: 700;
+        }
+      }
+    }
+  }
+}
```
Comment (dwi2) [inline src/style/Create_Member.css]: need a new line

File: src/style/PhotoList.css [hunk 5]
```diff
@@ -3,13 +3,14 @@
   grid-template-rows: 50px auto 50px;
   height: 100vh;
   overflow-x: hidden;
-  overflow-y: scroll;
+  /* overflow-y: scroll; */
```
Comment (dwi2) [inline src/style/PhotoList.css]: I'll suggest we directly remove this line.

File: src/style/App.css [hunk 6]
```diff
@@ -29,15 +30,21 @@ body, html{
...
+    border-color: #999;
+    color: #999;  
+    
+  }
+
   &:focus{
     outline: none;
   }
-}
+}
```
Comment (dwi2) [inline src/style/App.css]: need a new line.

File: src/components/Create_Group.js (hunk 1 above)
Comment (AsukaCHikaru) [reply src/components/Create_Group.js]: Ok

**Comment by AsukaCHikaru** [comment]
Updated

**Pull request merged**